Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -120,9 +124,20 @@ public PythonDeephavenSession(
         }
         scriptFinder = null;
 
+        registerJavaExecutor();
         publishInitial();
     }
 
+    private void registerJavaExecutor() {
+        // TODO (deephaven-core#4040) Temporary exec service until we have cleaner startup wiring
+        try (PyModule pyModule = PyModule.importModule("deephaven.server.executors");
+                final PythonDeephavenThreadsModule module = pyModule.createProxy(PythonDeephavenThreadsModule.class)) {
+            ExecutorService executorService = Executors.newFixedThreadPool(1);
+            module._register_named_java_executor("serial", executorService::submit);
+            module._register_named_java_executor("concurrent", executorService::submit);
```

Comment:
Handle possible duplication exception from Python?

Reply:
Given that these are the only two calls here, back to back, I dont think this is possible? - or if it is, it should just throw and be handled above here as "everything is broken, shut down", etc?